integrations/pom.xml

```diff
@@ -266,7 +266,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>jackson2-api</artifactId>
-      <version>2.8.11.2</version>
+      <version>2.9.9.1</version>
```

Comment:
We probably want to move to 2.9.10 really, is that possible?

Reply:
I just bumped those versions with: 
- https://github.com/jenkinsci/configuration-as-code-plugin/pull/1118/commits/4a193f04b8a54a23dedb1fa6cb01dc368602e9aa